VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoMatches.tsx

```diff
@@ -52,56 +35,16 @@ const PrescriptionHistoryNoMatches: FC<PrescriptionHistoryNoMatchesProps> = ({ c
       return getFilteredNoMatch()
     }
 
-    switch (currentTab) {
-      case PrescriptionHistoryTabConstants.ALL:
-        return (
-          <>
-            <TextView textAlign={'center'} variant="MobileBodyBold">
-              {t('prescription.history.empty.title')}
-            </TextView>
-            <TextView
-              pt={theme.dimensions.condensedMarginBetween}
-              textAlign={'center'}
-              variant="MobileBody"
-              accessibilityLabel={a11yLabelVA(t('prescription.history.empty.message'))}>
-              {t('prescription.history.empty.message')}
-            </TextView>
-          </>
-        )
-      case PrescriptionHistoryTabConstants.PENDING:
-        return (
-          <>
-            <TextView textAlign={'center'} variant="MobileBodyBold">
-              {t('prescription.history.empty.pending.title')}
-            </TextView>
-            <TextView
-              pt={theme.dimensions.condensedMarginBetween}
```

Comment:
same thing with strings, can't some be removed from common translation file?

Reply:
Removed prescription.history.empty.pending.* and prescription.history.empty.tracking.*